tests/spread/store/resources/task.yaml

```diff
@@ -64,11 +64,11 @@ execute: |
 
   # upload and verify the file resource
   upload_revision_number=$(charmcraft upload-resource $CHARM_DEFAULT_NAME example-file --format=json --file=resourcefile.txt --arch=amd64,riscv64 | jq -r '.revision')
-  revisions_json=$(charmcraft resource-revisions $CHARM_DEFAULT_NAME example-file --format=json)
-  upload_revision=$(echo "$revisions_json" | jq -c ".[] | select(.revision == $upload_revision_number)" )
+  charmcraft resource-revisions $CHARM_DEFAULT_NAME example-file --format=json > file_revisions.json
```

Comment:
Do you need to delete file_revisions.json when you're done?

Reply:
Nah, the directory gets deleted when the test is cleaned up

Reply:
Much better, thanks!